### DISCUSSION TITLE
Shelly MINI1PMG3 Tasmota software in german ?

### DISCUSSION THREAD

**Discussion by coolman111** [arendst/Tasmota#22666] [Q&A]
I installed Tasmota 14.4.1 successfully on my new Shelly device:
Device name: Shelly 1PM Mini Gen3
Device model: S3SW-001P8EU
Tasmota name: Shelly MINI1PMG3

I installed tasmota zip file 
mgos32-to-tasmota32-Mini1PMG3.zip   (version 14.4.1)
by manually upload in the Shelly Web Interface.
I did all the steps written on website
https://github.com/tasmota/mgos32-to-tasmota32

Finally everything is ok, but it is the English version.
The OTA update path is:
http://ota.tasmota.com/tasmota32/release/tasmota32c3.bin

I cannot find any file like "tasmota32c3-DE.bin" for a german language version.
How can I OTA update the firmware to the german language ?

At the  Tasmota release page http://ota.tasmota.com/tasmota32/release/
there are onlay files like "tasmota32-DE.bin"
Can I update to this "tasmota32-DE.bin" file or would I brick my device ?
(the "c3" is missing for Germany)

Best regards and hope to get some help
coolman111

**Comment by sfromis** [comment]
That would be bricking the device, different hardware architecture, but you can do a custom build. Not hard to do.

**Reply by coolman111** [reply]
Thank you for the information and warning.
I don't have any experience in compiling and making custom builds for Tasmota.
Perhaps language versions will be supported for the "c3"  architecture in near future, so I will wait.
Understanding english language is not a problem for me.

**Reply by coolman111** [reply]
@sfromis:
concerning my Shelly 1PM Mini G3 (see my post of Dec 18,2024):
can I update from 14.4.1 to 14.5.0 by simply clicking the Start Upgrade button of OTA update?
The OTA Url is: http://ota.tasmota.com/tasmota32/release/tasmota32c3.bin
I SHOULD NOT DO  the Auto Configuration or anything else once again, I suppose ?

**Reply by sfromis** [reply]
That should indeed be unproblematic, without any reconfiguration afterwards.

**Comment by HofeBY2** [comment]
@coolman111

Eine deutsche Anleitung zum selbst compilieren finden Sie unter

github.com/HofeBY
